ui/src/app/pages/analysis/leonardo-app-launcher.tsx

```diff
@@ -222,17 +245,20 @@ interface State {
   progressComplete: Map<Progress, boolean>;
 }
 
+const terminalName = '1'
```

Comment:
I would add a comment, not clear why we always choose this as the terminal name.